lib/presentation/tokens/spacing.dart

```diff
@@ -0,0 +1,12 @@
+class Spacing {
+  static const fourPX = 4.0;
+  static const eightPX = 8.0;
+  static const twelvePX = 12.0;
+  static const sixteenPX = 16.0;
+  static const twentyPX = 20.0;
+  static const twentyFourPX = 24.0;
+  static const twentyEightPX = 28.0;
+  static const thirstyTwoPX = 32.0;
+  static const thirstySixPX = 36.0;
+  static const fortyPX = 40.0;
+}
```

Comment:
<img width="680" alt="image" src="https://github.com/benikgalstyan/askm/assets/105119340/b48fc0c4-b19d-4542-a3c4-00b18ae502c4">

go through the design and put all the used spaces, paddings and radiuses into reusable variables like here